### PR TITLE
WiseCleaner.com.WiseCare365 version 5.7.1

### PR DESCRIPTION
**Pull request by Creator54**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/19513)

### CODE REVIEW

File: manifests/w/WiseCleaner/com/WiseCare365/5.7.1.573/WiseCleaner.com.WiseCare365.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: WiseCleaner.com.WiseCare365
+PackageVersion: 5.7.1.573
+PackageLocale: en-US
+Publisher: WiseCleaner.com
```
Comment (OfficialEsco) [inline manifests/w/WiseCleaner/com/WiseCare365/5.7.1.573/WiseCleaner.com.WiseCare365.locale.en-US.yaml:7]:
```suggestion
Publisher: WiseCleaner.com, Inc.
```

File: manifests/w/WiseCleaner/com/WiseCare365/5.7.1.573/WiseCleaner.com.WiseCare365.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: WiseCleaner.com.WiseCare365
+PackageVersion: 5.7.1.573
```
Comment (OfficialEsco) [inline manifests/w/WiseCleaner/com/WiseCare365/5.7.1.573/WiseCleaner.com.WiseCare365.installer.yaml:5]:
```suggestion
PackageIdentifier: WiseCleaner.WiseCare365
PackageVersion: 5.7.1
```

File: manifests/w/WiseCleaner/com/WiseCare365/5.7.1.573/WiseCleaner.com.WiseCare365.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: WiseCleaner.com.WiseCare365
+PackageVersion: 5.7.1.573
```
Comment (OfficialEsco) [inline manifests/w/WiseCleaner/com/WiseCare365/5.7.1.573/WiseCleaner.com.WiseCare365.locale.en-US.yaml:5]:
```suggestion
PackageIdentifier: WiseCleaner.WiseCare365
PackageVersion: 5.7.1
```

File: manifests/w/WiseCleaner/com/WiseCare365/5.7.1.573/WiseCleaner.com.WiseCare365.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: WiseCleaner.com.WiseCare365
+PackageVersion: 5.7.1.573
```
Comment (OfficialEsco) [inline manifests/w/WiseCleaner/com/WiseCare365/5.7.1.573/WiseCleaner.com.WiseCare365.yaml:5]:
```suggestion
PackageIdentifier: WiseCleaner.WiseCare365
PackageVersion: 5.7.1
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Remember to change folder and file names.

**Comment by OfficialEsco** [comment]
Not sure why the bot restarted, as far as i see the Filename is still wrong

Current `**WiseCleaner.com**.WiseCare365.installer.yaml`
Exepected `**WiseCleaner**.WiseCare365.installer.yaml`

**Comment by Creator54** [comment]
> Not sure why the bot restarted, as far as i see the Filename is still wrong
> 
> Current `**WiseCleaner.com**.WiseCare365.installer.yaml`
> Exepected `**WiseCleaner**.WiseCare365.installer.yaml`

sorry for that pushing a commit to fix it.

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**